src/templates/security-scheme-template.js

```diff
@@ -192,14 +192,15 @@ function oAuthFlowTemplate(flowName, clientId, clientSecret, apiKeyId, authFlow)
   let tokenUrl = authFlow.tokenUrl;
   let refreshUrl = authFlow.refreshUrl;
   const isUrlAbsolute = (url) => (url.indexOf('://') > 0 || url.indexOf('//') === 0);
+  const serverUrlObj = new URL(this.selectedServer.computedUrl);
   if (refreshUrl && !isUrlAbsolute(refreshUrl)) {
-    refreshUrl = `${this.selectedServer.computedUrl}/${refreshUrl.replace(/^\//, '')}`;
```

Comment:
I'm actually thinking for these three that we should follow the same pattern as the rest of the web and not custom logic here. The best way to do that would be:
```js
refreshUrl = new URL(refreshUrl, this.selectedServer.computedUrl);
```

And if we do this we can remove the method `isUrlAbsolute` because the `URL` takes care of that part too

Reply:
Nice, I didn't know about this constructor, let me update my PR!